### PR TITLE
Update requirements and update doc for SC getting started

### PR DESCRIPTION
**Pull request by sydhds**

### CODE REVIEW

File: docs/technical-doc/dummy-network-generation.rst [hunk 1]
```diff
@@ -45,9 +45,10 @@ Setup your node to use the secret you just generated as its public key and staki
 
     {"secret_key":"SECRETK","public_key":"PUBK"}
 
- * modify the file `massa-node/base_config/initial_ledger.json` :
+ * clear and modify the file `massa-node/base_config/initial_ledger.json` :
 
  .. code-block:: javascript
+
```
Comment (AurelienFT) [inline docs/technical-doc/dummy-network-generation.rst:51]: Can you edit this to have the actual format please ?

Comment (sydhds) [reply docs/technical-doc/dummy-network-generation.rst:51]: Done

**Review by AurelienFT — APPROVED** [review]
Thank you for the precisions on the docs !

**Comment by AurelienFT** [comment]
bors merge

**Pull request merged**